src/main/java/io/wisetime/connector/jira/JiraDao.java

```diff
@@ -134,14 +135,20 @@ boolean hasConfiguredTimeZone() {
         );
   }
 
-  List<Issue> findIssuesOrderedById(final long startIdExclusive, final int maxResults) {
-    return query().select("SELECT jiraissue.id, project.pkey, jiraissue.issuenum, jiraissue.summary, jiraissue.timespent "
+  List<Issue> findIssuesOrderedById(final long startIdExclusive, final int maxResults, final String... projectKeys) {
+    String query = "SELECT jiraissue.id, project.pkey, jiraissue.issuenum, jiraissue.summary, jiraissue.timespent "
         + "FROM project INNER JOIN jiraissue ON project.id = jiraissue.project "
-        + "WHERE jiraissue.id > ? ORDER BY ID ASC LIMIT ?;")
-        .params(
-            startIdExclusive,
-            maxResults
-        )
+        + "WHERE jiraissue.id > :startIdExclusive ";
+
+    if (ArrayUtils.isNotEmpty(projectKeys)) {
+      query += "AND project.pkey in (:projectKeys) ";
```

Comment:
Don't we need to handle casing (e.g. WT == wt)? Maybe not.

Reply:
Better be safe. I'm not sure if Jira project keys are case sensitive.